manifests/failover.pp

```diff
@@ -11,11 +11,13 @@
   $load_split           = '128',
   $load_balance         = '3',
   $omapi_key            = '',
-  $dhcp_dir             = $dhcp::params::dhcp_dir,
```

Comment:
This class parameter was only just added, and there hasn't been a release with it in.